src/app/app-testing/app-testing.component.html

```diff
@@ -0,0 +1 @@
+<p>app-testing works!</p>
```

Comment:
We should probably display something more useful on the actual component 😃

Reply:
Maybe later, lol